gui/wxpython/modules/import_export.py

```diff
@@ -644,13 +647,7 @@ def OnRun(self, event):
             if ext and layer.rfind(ext) > -1:
                 layer = layer.replace("." + ext, "")
             if "|" in layer:
-                layer, geometry = layer.split("|", 1)
-            else:
-                geometry = None
-
-                # TODO: v.import has no geometry option
-                # if geometry:
-                #    cmd.append('geometry=%s' % geometry)
+                layer, _ = layer.split("|", 1)
```

Comment:
You are redefining `_` here, that's why you get the error...

Reply:
Thanks for pointing that out! I've fixed it now